README_GPDB_ru.md

```diff
@@ -0,0 +1,207 @@
+```
+
+для ALT Linux p10
+```
+apt-get update
+apt-get install git gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel libyaml-devel zlib-devel 
+```
+
+- Добавить в переменную окружения PATH путь к pg_config Greenplum
+
+- Скачать репозиторий
+```
+git clone https://github.com/arenadata/pgbackrest -b 2.50-ci
+```
+
+- Перейти в каталог с исходным кодом
+```
+cd pgbackrest/src/
+```
+
+- Запустить скрипт конфигурирования
+
+Без параметров будут включены оптимизация (-O2) и векторизация вычисления контрольных сумм страниц.
+```
+./configure
+```
+
+Конфигурирование для отладки рекомендуется выполнять командой
+```
+CFLAGS="-O0 -g3" ./configure --disable-optimize --enable-test
+```
+
+- Собрать
+
+Команда для сборки с использованием всех доступных ядер и без отображения исполняемых команд
+```
+make -j`nproc` -s
+```
+
+В результате в текущем каталоге появится исполняемый файл с именем `pgbackrest`.
+
+- Установить
+
+```
+sudo make install
+```
+По умолчанию исполняемый файл `pgbackrest` будет размещен в каталог `/usr/local/bin`. Это единственное действие, которое выполняется на этапе установки.
+
+
+# 3. Настройка
+- Создать каталоги для размещения бэкапа и логов. Пусть для теста это будут `/tmp/backup` и `/tmp/backup/log`.
+```
+mkdir -p /tmp/backup/log
+```
+
+- Создать конфигурационный файл
+
+Для **стандартного демонстрационного кластера**, созданного с `DATADIRS=/tmp/gpdb`, этот файл будет выглядеть так:
+```
+[seg-1]
+pg1-path=/tmp/gpdb/qddir/demoDataDir-1
+pg1-port=6000
+
+[seg0]
+pg1-path=/tmp/gpdb/dbfast1/demoDataDir0
+pg1-port=6002
+
+[seg1]
+pg1-path=/tmp/gpdb/dbfast2/demoDataDir1
+pg1-port=6003
+
+[seg2]
+pg1-path=/tmp/gpdb/dbfast3/demoDataDir2
+pg1-port=6004
+
+[global]
+repo1-path=/tmp/backup
+log-path=/tmp/backup/log
+start-fast=y
+fork=GPDB
+```
+
+В дальнейших примерах команд предполагается, что конфигурационный файл имеет стандартное имя - `/etc/pgbackrest.conf`. Если требуется использовать другой файл, то его имя можно передать через параметр `--config`.
+
+Так как данная версия pgBackRest может применяться для бэкапа как PostgreSQL, так и Greenplum, следует указать в параметре `fork`, бэкап какой СУБД выполняется. Описание остальных параметров можно найти в [документации](https://pgbackrest.org/configuration.html) или в `build/help/help.xml`.
+
+- Создать и инициализировать для координатора и каждого первичного сегмента каталоги, в которых будут храниться файлы для восстановления
+```
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" pgbackrest --stanza=seg$i stanza-create
+done
+```
+Под термином "stanza" понимается "строфа" в конфигурационном файле
+
+- Настроить Greenplum
+```
+gpconfig -c archive_mode -v on
+gpconfig -c archive_command -v "'PGOPTIONS=\"-c gp_session_role=utility\" /usr/local/bin/pgbackrest --stanza=seg%c archive-push %p'" --skipvalidation
+gpstop -ar
+```
+
+- Установить расширение gp_pitr 
+```
+create extension gp_pitr;
```

Comment:
All commands above are for shell. This one is for 'psql'. I think it should be highlighted to avoid somebody trying to enter it into the shell.

And there are several similar cmds below. The same comment for them.